db/migrate/20180323134330_chenges_type_of_amount_in_categories_and_transactions.rb

```diff
@@ -0,0 +1,7 @@
+class ChengesTypeOfAmountInCategoriesAndTransactions < ActiveRecord::Migration[5.1]
+  def change
+    remove_column :categories, :amount, :string
+    add_column :categories, :amount, :decimal, precision: 8, scale: 2
+    change_column :transactions, :amount, :decimal, precision: 8, scale: 2
```

Comment:
@AleksSenkou yes, I tried so do, but was problems. I thought that here I could use  recreate after removing, because all categories have `amount: 0`.

Reply:
@marderer let's keep safe policy while data migration